java/tracing/src/test/scala/org/typelevel/otel4s/java/trace/SpanNode.scala

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2022 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.otel4s.java.trace
+
+import io.opentelemetry.sdk.trace.data.SpanData
+
+import scala.concurrent.duration._
+
+// Tree-like representation of a span
+final case class SpanNode(
```

Comment:
Where does this get used?

Reply:
In TracerSuite: https://github.com/typelevel/otel4s/blob/9ff15db3ee11522b783d5e9dc8401396782011df/java/tracing/src/test/scala/org/typelevel/otel4s/java/trace/TracerSuite.scala#L455-L465

Reply:
Eek.  I didn't notice that it was test code.  I'm generally trying to keep non-Otel concepts out of the core API.